src/components/EmailForm/index.jsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from "react"
+import { Link } from "react-router-dom"
+import InputBox from "../InputBox"
+
+function EmailForm(){
+
+  const [email, setEmail] = useState('')
+  const [subject, setSubject] = useState('')
+  const [message, setMessage] = useState('')
+  const [confirm, setConfirm] = useState('')
+  const emailChange = (e) => {setEmail(e.target.value)}
+  const subjectChange = (e) => {setSubject(e.target.value)}
+  const handleSubmit = (e) => {
+      e.preventDefault()
+      getData()
+  }
+    
+  function getData (){
```

Comment:
You should rename this function to something more descriptive - just sendEmail would do